### PR TITLE
Adopt dotenv

### PR DESCRIPTION
**Pull request by lorentzlasson**
Just a suggestion, wont insist :) 

---

Tested on devices 
- [ ] Desktop 💻
- [ ] Mobile 📱

Tests  
- [x] All tests are running ✔️
- [ ] Test are updated 🧪
- [ ] Code Review 👩‍💻
- [ ] QA 👌

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,8 +80,8 @@
     "@types/node": "^16.0.0",
     "@types/request-promise-native": "^1.0.17",
     "@types/sinon": "^7.5.2",
-    "chai-datetime": "^1.7.0",
     "chai": "^4.2.0",
+    "chai-datetime": "^1.7.0",
```
Comment (lorentzlasson) [inline package.json:84]: Unrelated - guess it was auto sorted when the new package was added

File: README.md [hunk 2]
```diff
@@ -277,11 +278,9 @@ The other configuration variables are only needed to run specific parts of the c
 Let's run the service, using our locally-running MySQL instance (either native or in Docker):
 
 ```
-DB_NAME='EffektDonasjonDB_Local' DB_USER='root' DB_PASS='effekt' npm start
+npm start
 ```
 
-(Depending on your operation system, setting environment variables may be a different process. Check out the procedure for [windows](docs/win_environment_variables.md) and [macos / linux](nix_environment_variables.md) respectively).
```
Comment (lorentzlasson) [inline README.md:283]: AFAIK there shouldn't be any platform specifics to worry about if we use dotenv

**Review by adamalfredsson — APPROVED** [review]
Good suggestion I think!

**Comment by fellmirr** [comment]
Looks very good :+1:

**Pull request merged**